projects/plugins/jetpack/_inc/lib/core-api/wpcom-fields/post-fields-publicize-connections.php

```diff
@@ -120,6 +120,12 @@ private function post_connection_schema() {
 					'readonly'    => true,
 				),
 				'display_name'    => array(
+					'description' => __( 'Usually username of the connected account', 'jetpack' ),
```

Comment:
Is this right? Is it normally the username?

Reply:
In the case of Facebook, it's the Facebook page's name, isn't it?

Reply:
Yeah, my point being that the display name is usually *not* the username